content/kubernetes/re-clusters/production-recommendations.md

```diff
@@ -0,0 +1,83 @@
+
+```yaml
+apiVersion: app.redislabs.com/v1
+kind: RedisEnterpriseCluster
+metadata:
+  name: example-redisenterprisecluster
+  labels:
+    app: redis-enterprise
+spec:
+  nodes: 3
+  nodeSelector:
+    cloud.google.com/gke-nodepool: pool1
+```
+## Eviction thresholds
+
+Kubernetes uses [node-pressure eviction](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/) to terminate pods and reclaim node resources. Best practice for production Redis Enterprise deployments: 
+
+* Set a high [soft eviction threshold](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#soft-eviction-thresholds), which will change the node condition early and notify the administrator.
+* Set the `eviction-soft-grace-period` high enough for the administrator to scale the deployment up or out when needed.
+* Set the `eviction-max-pod-grace-period` high enough to allow Redis Enterprise pods to migrate the database before being evicted.
+
+## Monitor resources
+
+There are two [node conditions](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#node-conditions) that are `True` if your [eviction threshold](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#eviction-thresholds) has been met and a pod eviction is likely:  `MemoryPressure` and `DiskPressure`.
+
+Use the `kubectl get nodes` command to monitor these conditions.
+
+```yaml
+> kubectl get nodes -o jsonpath='{range .items[*]}name:{.metadata.name}{"\t"}MemoryPressure:{.status.conditions[?(@.type == "MemoryPressure")].status}{"\t"}DiskPressure:{.status.conditions[?(@.type == "DiskPressure")].status}{"\n"}{end}'
+name:gke-55d1ac88-213c	MemoryPressure:False	DiskPressure:False
+name:gke-55d1ac88-vrpp	MemoryPressure:False	DiskPressure:False
+name:gke-7253cc19-42g0	MemoryPressure:False	DiskPressure:False
+```
+
+## Resource limits
+
+The Redis Enterprise operator is set with resource limits and requests by default. You can find the recommended settings in the []`operator.yaml`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/operator.yaml) file.
+
+If you create a [ResourceQuota](https://kubernetes.io/docs/concepts/policy/resource-quotas/) on the namespace containing the RedisEnterpriseCluster, be careful applying the quotas to ConfigMaps, as it can reach the limit with as little as one ConfigMap.
+
+## Pod security
+
+[PodSecurityPolicy](https://kubernetes.io/docs/concepts/security/pod-security-policy/) has been deprecated for Kubernetes versions 1.21 and after and is invalid for versions 1.25 and after. Please migrate to using the [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/).
+
+There are three Pod Security Admission levels: `restricted`, `baseline`, and `privileged`. Redis Enterprise pods require the `privileged` level.
```

Comment:
Here we can add a recommendation, that if Pod Security Admission is enabled, it's recommended to run the REC in a dedicated namespaces without other workloads (because that namespace is required to allow `privileged` pods like the REC pods...).